src/ui/layout/responsive/building-adaptive-apps.md

```diff
@@ -397,6 +403,16 @@ return Padding(
 );
 ```
 
+Use `Theme.of(context).platform` for theming and
+design choices, like what kind of switches to show
+and general Cupertino/Material adaptions.
+It also helps an app work well on mobile web.
+
+Use `defaultTargetPlatform` if you're implementing
+platform logic in flutter/widgets, or are otherwise
```

Comment:
What is "flutter/widgets" here? It sounds like it's referring to the implementation of the [`flutter/widgets`](https://api.flutter.dev/flutter/widgets/widgets-library.html) library. I think we should drop the mention if so.

Reply:
Hmm, taking a look at this whole paragraph, I think it's specific to implementers of Flutter as others should generally use `Theme.of(context).platform`.